scrapy/templates/project/module/middlewares.py.tmpl

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+# Define here the models for your spider middleware
+#
+# See documentation in:
+# http://doc.scrapy.org/en/latest/topics/spider-middleware.html
+
+
+class ${ProjectName}SpiderMiddleware(object):
+
```

Comment:
What do you think of adding `from_crawler` and maybe registering a signal on `spider_opened`, (like in [`OffsiteMiddleware`](https://github.com/scrapy/scrapy/blob/master/scrapy/spidermiddlewares/offsite.py)) to show best practices?